src/TMS/Translations.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/* Copyright (c) 2018 Nils Haagen <nils.haagen@concepts-and-training.de> */
+
+namespace ILIAS\TMS;
+
+
+/**
+ * This is the interface to a decorator for accumulating translations.
+ */
+interface Translations  {
```

Comment:
Ich finde `Translations` nicht wirklich passend. Hier wird ja nicht wirklich was übersetzt. Irgendwie ist das hier eher eine Auflösung von abstrakten Labels zu verständlichem Text. Ein besserer Name fällt mir gerade aber auch nicht wirklich ein. Vllt. `LabelResolution`? Oder `TextResolver`?

Reply:
The "Text-Revolver", an early form of the "Machine-Gun-conversation" :)